engines/router/missionctl/server/grpc/server.go

```diff
@@ -0,0 +1,157 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/caraml-dev/turing/engines/router/missionctl"
+	"github.com/caraml-dev/turing/engines/router/missionctl/errors"
+	"github.com/caraml-dev/turing/engines/router/missionctl/instrumentation/metrics"
+	"github.com/caraml-dev/turing/engines/router/missionctl/instrumentation/tracing"
+	"github.com/caraml-dev/turing/engines/router/missionctl/log"
+	"github.com/caraml-dev/turing/engines/router/missionctl/log/resultlog"
+	"github.com/caraml-dev/turing/engines/router/missionctl/turingctx"
+	upiv1 "github.com/caraml-dev/universal-prediction-interface/gen/go/grpc/caraml/upi/v1"
+	"github.com/gojek/fiber"
+	fibergrpc "github.com/gojek/fiber/grpc"
+	"github.com/opentracing/opentracing-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/reflection"
+)
+
+const turingReqIDHeaderKey = "Turing-Req-ID"
```

Comment:
Minor thing - but should we move this to a `constants.go` or something so it can be reused for all handlers?

Reply:
i thought of similar but it will be a standalone file+package with this one variable. Will put it in `server/constant` for now